github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml

```diff
@@ -167,6 +167,13 @@ plugins:
     - owners-label
     - trigger
 
+  kubevirt/enhancements:
+    plugins:
+    - approve
+    - lgtm
+    - owners-label
+    - trigger
```

Comment:
Apparently also needs the `release-note` plugin.

```
{"component":"checkconfig","file":"sigs.k8s.io/prow/cmd/checkconfig/main.go:91","func":"main.reportWarning","level":"warning","msg":"the following orgs or repos forbid the do-not-merge/release-note-label-needed label for merging but do not enable the release-note plugin: [repo: kubevirt/enhancements]","severity":"warning","time":"2025-01-09T14:39:53Z"}
```

Reply:
sec